functions/New-DbaLogin.ps1

```diff
@@ -493,12 +508,31 @@ function New-DbaLogin {
                                 }
                             }
                         }
+
+                        #Process the MustChangePassword property
+                        if ($currentPasswordMustChange -ne $newLogin.MustChangePassword) {
+                            try {
+                                $newLogin.ChangePassword($SecurePassword, $true, $true)
+                                Write-Message -Level Verbose -Message "Login $loginName has been marked as must change password."
+
+                                # We need to refresh login after ChangePassword. Otherwise, MustChangePassword will appear as False
+                                $server.Logins[$loginName].Refresh()
+                            } catch {
+                                Write-Message -Level Verbose -Message "Failed to marked as must change password in $loginName on $instance using SMO."
+                            }
+                        }
+
                         #Display results
                         # If we ever used T-SQL, the smo is some times not up to date and should be refreshed
                         if ($usedTsql) {
                             $server.Logins.Refresh()
                         }
-                        Get-DbaLogin -SqlInstance $server -Login $loginName
+
+                        if ($loginType -eq 'SqlLogin') {
+                            Get-DbaLogin -SqlInstance $server -Login $loginName | Select-Object ComputerName, InstanceName, SqlInstance, Name, LoginType, CreateDate, LastLogin, HasAccess, IsLocked, IsDisabled, MustChangePassword
```

Comment:
We should return a Login object in all cases. Select-Object would create a custom object instead. Did you want to use Select-DefaultView

Reply:
Excellent catch! 
Changed. Thanks.

Reply:
Not a problem. I'm still wondering though why SqlLogin needs a special treatment here.

Reply:
It's just my suggestion and how I would like it to have it. 

Windows login can't be created with must change pwd flag. Thus, don't make sense to output. 
However, if we are creating one (or more) SQL login and we specify the flag I find it useful to know if it needs to change the password. I see it like we have the isDisabled.

Reply:
I see this handled better by `Get-DbaLogin` than with the new function. Just add this logic to the Get command.

Reply:
The same reason could be said on the output itself of `Get-DbaLogin`..so no real reason to not just do it there.

Reply:
- Rollback the change of the output on the New-DbaLogin
- Added MustChangePassword to the default output of Get-DbaLogin
- Took the chance to add a new switch parameter -MustChangePassword that will just output logins that need to change password.

Do you agree?